### PR TITLE
Remove qwery from commercial hosted gallery

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

For commercial hosted gallery:

- remove qwery
- remove $
- remove bonzo 
- remove $$ (superfluous once bonzo removed)

I decided not to convert this to TS as its a very large file with a lot of dependencies and dynamic binding to unravel.

The reward / effort required would be small for something we will probably replace in future.

Part of #23740

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/hosted/gallery.js [hunk 1]
```diff
@@ -317,29 +308,33 @@ class HostedGallery {
...
+            this.$images.forEach((image, index) => {
                 const opacity = ((progress - index + 1) * 16) / 11 - 0.0625;
-                bonzo(image).css('opacity', Math.min(Math.max(opacity, 0), 1));
+                image.style.setProperty('opacity', Math.min(Math.max(opacity, 0), 1));
             });
 
-            bonzo(this.$border).css('transform', `rotate(${deg}deg)`);
-            bonzo(this.$border).css('-webkit-transform', `rotate(${deg}deg)`);
+            this.$border.style.setProperty('transform', `rotate(${deg}deg)`);
+            this.$border.style.setProperty('-webkit-transform', `rotate(${deg}deg)`);
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/hosted/gallery.js:317]: I think we can drop this, there's now over 96% support for the non-prefixed version.

https://caniuse.com/transforms2d

**Review by mxdvl — COMMENTED** [review]
Wow, this is a lot of Javascript to handle animations. I think before we approve this, it would be worth adding an link example to such a gallery.

File: static/src/javascripts/projects/commercial/modules/hosted/gallery.js (hunk 1 above)
Comment (arelra) [reply static/src/javascripts/projects/commercial/modules/hosted/gallery.js:317]: Opera mini 😢 ... jk

**Comment by arelra** [comment]
> Wow, this is a lot of Javascript to handle animations. I think before we approve this, it would be worth adding an link example to such a gallery.

Yes its surprising they rolled their own slideshow rather than using a lib.

I previously added an example of a hosted gallery to `gallery.spec.js` [here](https://github.com/guardian/frontend/blob/4898c993098b8abac4c800e5a3a510275c60cbe3/static/src/javascripts/projects/commercial/modules/hosted/gallery.spec.js#L8)

For reference:

https://www.theguardian.com/advertiser-content/microsoft-ai-for-earth/meet-the-changemakers

**Comment by mxdvl** [comment]
The fact that you can scroll and stop mid-way between two slides is actually pretty jarring, and not a great experience:

cross-fade between two picture, making them indistinguishable

**Comment by arelra** [comment]
> The fact that you can scroll and stop mid-way between two slides is actually pretty jarring, and not a great experience:
> 
> cross-fade between two picture, making them indistinguishable

Someone had lots of fun with opacity

**Comment by arelra** [comment]
> The fact that you can scroll and stop mid-way between two slides is actually pretty jarring, and not a great experience:
> 
> cross-fade between two picture, making them indistinguishable

Not high priority but once we migrate this out we should replace it with something low-maintenance and off the shelf.

**Review by mxdvl — APPROVED** [review]

**Pull request merged**